kolibri/plugins/audio_mp3_render/kolibri_plugin.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+"""
+Copy and modify this code for your own plugin.
+"""
+from __future__ import absolute_import, print_function, unicode_literals
+
+import logging
+
+from kolibri.plugins.base import KolibriFrontEndPluginBase
+from kolibri.plugins.hooks import CONTENT_RENDERER_ASYNC, FRONTEND_PLUGINS
+
+logger = logging.getLogger(__name__)
+
+
+class KolibriAudioMP3RenderFrontEnd(KolibriFrontEndPluginBase):
+    """
+    The base learn code for the learn page.
+    """
+    entry_file = "assets/src/render_audio_mp3_module.js"
+
+    events = {
+        "content_render:audio/mp3": "render"
```

Comment:
> What is less clear is if two different content playing plugins have both been registered at once. 

That's what I'm wondering about. If it doesn't make sense to have multiple plugins registered to handle the same content type then doing so should result in an error, maybe even at server start time. Otherwise the Learn page I think should gracefully handle this situation.